src/sandstorm/test-app/test-app.c++

```diff
@@ -151,6 +154,47 @@ public:
       response.setMimeType("text/html");
       response.initBody().setBytes(isPowerboxRequest ? *TEST_POWERBOX_HTML : *TEST_APP_HTML);
       return kj::READY_NOW;
+    } else if (path == "shutdown" || path == "/shutdown/") {
+      auto staticFd = raiiOpen("/var/www/index.html", O_RDWR|O_CREAT);
+      auto data = TEST_SHUTDOWN_HTML.get();
+      KJ_SYSCALL(write(staticFd.get(), data.begin(), data.size()));
+      auto response = context.getResults();
+      auto content = response.initContent();
+      content.setStatusCode(sandstorm::WebSession::Response::SuccessCode::OK);
+      content.setMimeType("text/plain");
+
+      // After sending the return message, shut down the connection and then
+      // exit.
+      //
+      // We need to wait for the return message to actually be sent,
+      // otherwise the supervisor will try to re-connect in order to retry
+      // the method call, which will start us back up again.
+      //
+      // Note that we need to call evalLater *twice* to make this happen,
```

Comment:
Is this a good use case for `evalLast()`?

Reply:
Good call, didn't know about that one. Done, and rebased on top of master.